### DISCUSSION TITLE
Request:  Floorplan graphics

### DISCUSSION THREAD

**Discussion by MenooMeow** [sysadminsmedia/homebox#380] [Ideas]
I'd really love to be able to upload a floor plan of my house where I can reference a location with a graphic.  Maybe the ability to superimpose simple polygons to indicate the general location of storage boxes or racks.

**Comment by supaeasy** [comment]
Maybe https://espresense.com/Floorplan-Creator/ is a start. It supports an export to yaml so you could add room plans and box locations as text in a x/y coordinate system. Yes, it is not ideal.
